planner/core/expression_rewriter_test.go

```diff
@@ -343,3 +343,45 @@ func (s *testExpressionRewriterSuite) TestIssue17652(c *C) {
 	tk.MustQuery("select ifnull(max(x), 0) from t").Check(
 		testkit.Rows("9999999703771440633"))
 }
+
+// https://github.com/pingcap/tidb/issues/8733#issuecomment-700572764
```

Comment:
The same with above. It's better to be the format like `TestExplain.... tests the ... See issue [the issue link] for more details.

Reply:
I will fix this.